### PR TITLE
Fix: update the init env doc

### PR DESCRIPTION
**Pull request by FogDong**
Signed-off-by: FogDong <dongtianxin.tx@alibaba-inc.com>

### Description of your changes

I have:

- [ ] Read and followed KubeVela's [contribution process](https://github.com/kubevela/kubevela/blob/master/contribute/create-pull-request.md).
- [ ] Update `sidebar.js` if adding a new page.
- [ ] Run `yarn start` to ensure the changes has taken effect.

### Special notes for your reviewer

cc @wonderflow

### CODE REVIEW

File: docs/case-studies/initialize-env.md [hunk 1]
```diff
@@ -54,20 +54,23 @@ metadata:
...
+        chart: kruise
+        version: 1.2.0
+        git:
+          branch: master
+        values:
+          featureGates: PreDownloadImageForInPlaceUpdate=true
   workflow:
     steps:
     - name: check-flux
       type: depends-on-app
```
Comment (wonderflow) [inline docs/case-studies/initialize-env.md:76]: does it still work?

File: docs/case-studies/initialize-env.md [hunk 2]
```diff
@@ -155,71 +192,143 @@ spec:
             - key: test-key
               path: test-key
 
+  policies:
+    - name: my-policy
+      properties:
+        clusters:
+        - local
+      type: topology
```
Comment (wonderflow) [inline docs/case-studies/initialize-env.md:205]: we should add apply-once to make the initialization match the real scenario.

File: docs/case-studies/initialize-env.md (hunk 1 above)
Comment (FogDong) [reply docs/case-studies/initialize-env.md:76]: yes

File: docs/case-studies/initialize-env.md [hunk 3]
```diff
@@ -36,8 +36,8 @@ If we want to use some CRD controller like [OpenKruise](https://github.com/openk
 
 We can directly use Application to initialize a kruise environment. The application below will deploy a kruise controller in cluster.
 
-We have to enable `fluxcd` in cluster since we use `Helm` to deploy kruise.
-We can use `depends-on-app` to make sure `fluxcd` is deployed before kruise.
+We have to enable `fluxcd` Addon in cluster since we use `Helm` to deploy kruise.
```
Comment (wonderflow) [inline docs/case-studies/initialize-env.md]:
```suggestion
We have to enable `fluxcd` addon in cluster since we use `Helm` to deploy kruise.
```

Comment (wonderflow) [reply docs/case-studies/initialize-env.md]: add a link to fluxcd addon doc.

**Review by wonderflow — APPROVED** [review]
LGTM

**Pull request merged**